com.onesignal.unity.ios/Editor/BuildPostProcessor.cs

```diff
@@ -111,6 +111,14 @@ public class BuildPostProcessor : IPostprocessBuildWithReport {
             // Add the service extension
             AddNotificationServiceExtension();
 
+            // Unity Tests
```

Comment:
This method's logic is mostly declarative vs imperative. Can we make a new method call something like `DisableBitcode()`?